.betterer.results

```diff
@@ -751,6 +751,10 @@ exports[`better eslint`] = {
       [0, 0, 0, "Unexpected any. Specify a different type.", "1"],
       [0, 0, 0, "Unexpected any. Specify a different type.", "2"]
     ],
+    "packages/grafana-data/src/types/templateVars.ts:5381": [
+      [0, 0, 0, "Unexpected any. Specify a different type.", "0"],
+      [0, 0, 0, "Unexpected any. Specify a different type.", "1"]
+    ],
```

Comment:
these `any`s were just copied across from `public/app/features/variables/types.ts`, and aren't actually new.

I think they can be removed, but it's a bunch more changes that I haven't fully explored yet.